modules/ROOT/pages/creating-an-odata-api-with-apikit.adoc

```diff
@@ -26,7 +26,7 @@ You install the APIKit OData Extension from Anypoint Studio as described in the
 . From the *Help* menu in Studio, select *Install New Software*.
 . Click *Add...*
 . In the *Name* field, type `APIkit for ODATA Update Site`.
-. In the *Location* field, type `+http://studio.mulesoft.org/s3/apikit-for-odata/+`.
+. In the *Location* field, type `+http://studio.mulesoft.org/s4/apikit-for-odata/+`.
```

Comment:
This isn't enough, for studio 7.4.x is ok but for previous versions it should use the the s3 url.

Reply:
Yes, that's OK. This is for studio 7.4 only.